concepts/slices/about.md

```diff
@@ -1,5 +1,31 @@
 # About
 
+Slices in Go are similar to lists or arrays in other languages. They hold a number of elements of a specific type (or interface).
+
+Slices in Go are based on arrays. Arrays have a fixed size. A slice, on the other hand, is a dynamically-sized, flexible view into the elements of an array.
+
+A slice is written like `[]T` with `T` being the type of the elements in the slice:
+
+```go
+var empty []int                 // an empty slice
+withData := []int{0,1,2,3,4,5}  // a slice pre-filled with some data
+```
+
+You can get/set an element at a given zero-based index using square-bracket notation:
```

Comment:
```suggestion
You can get or set an element at a given zero-based index using square-bracket notation:
```

I think we should prefer being more explicit about this rather than implying they are interchangeable or something like that.